addons/mail/static/src/discuss/call/common/thread_model_patch.js

```diff
@@ -160,12 +160,21 @@ const ThreadPatch = {
         return !this.store.rtc.state.isFullscreen && this.rtc_session_ids.length > 0;
     },
     focusAvailableVideo() {
-        if (this.isDisplayedInDiscussAppDesktop || !this.store.settings.useCallAutoFocus) {
+        if (
+            !this.store.settings.useCallAutoFocus ||
+            !(
+                this.store.env.services.ui.isSmall ||
+                this.store.rtc.state.isPipMode ||
+                (this.chat_window?.isOpen &&
+                    !this.store.meetingViewOpened &&
+                    !this.store.discuss.isActive)
+            )
```

Comment:
Runbot's red because `this.store.discuss` is `public_web` specific code, whereas the code here is in `common`.
Can define a function and patch it in `public_web` folder for the `discuss part`.